### PR TITLE
[DO NOT MERGE] - CircleCi

### PR DESCRIPTION
**Pull request by maneta**
**TL;DR: TO MERGE THIS WE WILL NEED TO CONFIGURE CIRCLE CI FIRST**

In this PR we introduce:

* CI/CD Circle CI Configuration.
* Correction to the Dockerfile.
* Correction to the docker `entrypoint` script.
* Remove a DB dummy test.

### CircleCI JOB

This Job run the tests inside containers on Circle CI. 
When the branch is **master** and  the tests are passing, the job will build an image with the new code, tag it with the commit SHA1 and push it to Dockerhub.

The next step is to create a Openshift Deployment configured with [Imagestreams](https://docs.openshift.com/enterprise/3.0/architecture/core_concepts/builds_and_image_streams.html#image-streams), so every time we push to master it will deploy the code to Openshift.

I have the CircleCI integration working on [my fork](https://github.com/maneta/project-X/tree/circle-ci).

I Could do the integration configuration on the barcelonapm repo but I would need admin rights over the repo.

Salut!

### CODE REVIEW

**Comment by maneta** [comment]
The generated image on Dockerhub: https://hub.docker.com/r/barcelonapm/project-x/tags/
